README.md

```diff
@@ -90,19 +90,19 @@ Then you can run:
 
 `npm run start`
 
-This will run webpack in the watch mode, so your changes will be saved in the build folder on the fly. After you're done making changes, run:
+This will run a webpack in the watch mode, so your changes will be saved in the build folder on the fly. After you're done making changes, run:
```

Comment:
```suggestion
This will run webpack in the watch mode, so your changes will be saved in the build folder on the fly. After you're done making changes, run:
```

Webpack is a tool, not an uncountable noun that requires `a`. Webpack is a concrete tool, not an unknown.

Reply:
What do you think this?

> This will run the webpack in the watch mode, so your changes will be saved in the build folder on the fly. After you're done making changes, run:

Reply:
Just remove the a, and leave the paragraph as it was.

Reply:
Done. I will PR soon.